### ISSUE TITLE
[Google Fit ]Support for recording user steps?

### ISSUE THREAD

**Issue by mychaelgo** [krokyze/FitKit#18]
Does this library support for recording user steps ?
Reference: https://developers.google.com/fit/scenarios/record-steps

**Comment by krokyze** [comment]
No, currently plugin only supports reading values from history.
